etc/rita.yaml

```diff
@@ -66,7 +66,7 @@ Bro:
 
 SafeBrowsing:
     APIKey: ""
-    Database: /usr/local/rita/safebrowsing_database
+    Database: $HOME/.rita/safebrowsing
```

Comment:
We need a place to store application data. A hidden directory in the home folder of the current user was deemed the most appropriate place. /var/lib was also heavily considered, but since the program is not run as a daemon or by root, /var/lib is not appropriate